packages/server/controllers/guilds.js

```diff
@@ -82,6 +82,41 @@ async function getGuildMembers(guildId) {
   return members;
 }
 
+async function getGuildsForUser(userId) {
+  try {
+    const user = await prisma.users.findFirst({
+      where: {
+        userId: userId,
+      },
+    });
+
+    if (!user) {
+      throw new Error(`User with ID ${userId} not found`);
+    }
+
+    const userGuilds = await prisma.guildMembers.findMany({
+      where: {
+        userId: userId,
+      },
+      include: {
+        guilds: {
+          select: {
+            avatar: true,
+            guildId: true,
+            name: true,
+            handler: true,
+          },
+        },
+      },
+    });
+
+    const guilds = userGuilds.map((userGuild) => userGuild.guilds);
+    return guilds;
+  } catch (error) {
+    throw new Error(`Error fetching guilds for user: ${error.message}`);
+  }
```

Comment:
Get rid of this catch block since the route handler already has a catch block to catch this error if it occurs.